docs/src/CommutativeAlgebra/ideals.md

```diff
@@ -214,9 +214,24 @@ leading_ideal(I::MPolyIdeal)
 
 #### Gröbner Bases over the integers
 
-#### ....
+Over the integers the coefficients of the polynomials 
+are not invertible, thus their handling when computing
+Gröbner bases and normal forms plays an important role. This is done when 
+computing strong Gröbner bases which ensure the following property: 
+For any element of an ideal its leading term is divisible by a leading term of an 
+element of a corresponding strong Gröbner basis.
 
+The textbook
 
+- [AL94](@cite)
+
+provides details on theory and algorithms as well as references.
```

Comment:
Hmm, why put this into a list, though?
```suggestion
The textbook [AL94](@cite) provides details on theory and algorithms as well as references.
```

Anyway, this is a matter of taste, I guess *shrug*